README.md

```diff
@@ -44,6 +44,8 @@ This plugin supports the following features:
 
 ## Installation
 
+These installation instructions assume you already have a [Mattermost instance](https://mattermost.com/download/) running [PostgreSQL](https://www.postgresql.org/):
```

Comment:
This comment doesn't explicitly state that only Postgres is supported.  It suggests the installation instructions are for a Postgres installation but could be interpreted as just a limitation of the instructions.  

I suggest a more epxlicit "Postgres required" be included.

Reply:
That's fair -- @crspeller, I copied this from the AI plugin. How have you been handling pushback on Postgres thus far? For both our plugins, perhaps we could tweak this language to be:

```suggestion
These installation instructions assume you already have a [Mattermost instance](https://mattermost.com/download/) running [PostgreSQL](https://www.postgresql.org/). Note that this plugin does not support MySQL.
```

Reply:
Pushback for support is documented in https://github.com/mattermost/mattermost-plugin-ai/issues/19

The tweak looks good for both plugins.